editor/components/post-publish-panel/postpublish.js

```diff
@@ -48,26 +49,35 @@ class PostPublishPanelPostpublish extends Component {
 	}
 
 	render() {
-		const { post, postType } = this.props;
+		const { isScheduled, post, postType } = this.props;
 		const viewPostLabel = get( postType, [ 'data', 'labels', 'view_item' ] );
 
+		const postPublishNonLinkHeader = isScheduled ?
+			<Fragment>{ __( 'is now scheduled. It will go live on' ) } <PostScheduleLabel />.</Fragment> :
+			__( 'is now live.' );
+		const postPublishBodyText = isScheduled ?
+			__( 'The post address will be:' ) :
+			__( 'What\'s next?' );
+
 		return (
 			<div className="post-publish-panel__postpublish">
 				<PanelBody className="post-publish-panel__postpublish-header">
-					<a href={ post.link }>{ post.title || __( '(no title)' ) }</a>{ __( ' is now live.' ) }
+					<a href={ post.link }>{ post.title || __( '(no title)' ) }</a> { postPublishNonLinkHeader }
 				</PanelBody>
 				<PanelBody>
-					<div><strong>{ __( 'What\'s next?' ) }</strong></div>
+					<div><strong>{ postPublishBodyText }</strong></div>
 					<input
 						className="post-publish-panel__postpublish-link-input"
 						readOnly
 						value={ post.link }
 						onFocus={ this.onSelectInput }
 					/>
 					<div className="post-publish-panel__postpublish-buttons">
-						<Button className="button" href={ post.link }>
-							{ viewPostLabel }
-						</Button>
+						{ ! isScheduled &&
```

Comment:
There is no Eslint rule for that but we usually wrap an element with parentheses when after && and is multiline.